redis/redis.go

```diff
@@ -0,0 +1,38 @@
+package redis
```

Comment:
You don't need this package at all. It's a level of abstraction too far. Just put the `disRedis.NewClient(…)` directly into the `DoGetRedis(…)` function in `service/initialise.go` instead. Similar for the `Checker`